xfields/beam_elements/beambeam3d.py

```diff
@@ -900,6 +980,7 @@ def slices_other_beam_pzeta_center(self, value):
         raise NotImplementedError(
             "Setting slices_other_beam_pzeta_center is not implemented yet")
 
+
```

Comment:
Remember to remove these commented lines

Reply:
these lines below werent put here by me should I still remove them?